db/migrate/20190726063735_add_position_id_to_players.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionIdToPlayers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :players, :position_id, :integer
```

Comment:
we'll want to add an index here ( and probably on other tables as well ) for performance, but that can wait.